core/src/main/scala/dagr/core/cmdline/parsing/CommandLineParser.scala

```diff
@@ -281,6 +282,14 @@ private[cmdline] class CommandLineParser[T](val targetClass: Class[T]) extends L
     builder.toString
   }
 
+  /** add extra whitespace after newlines for pipeline descriptions*/
+  private def formatLongDescription(description: String): String = {
+    val desc = description.stripMargin
+    desc
+      .dropWhile(_ == '\n')
```

Comment:
Does this mean that we can't use newlines to control formatting in the description?  E.g. if we want to render a set of bullets or something?